include/zephyr/bluetooth/mesh/access.h

```diff
@@ -672,6 +673,19 @@ struct bt_mesh_mod_id_vnd {
 	uint16_t id;
 };
 
+/** Mesh model ctx */
```

Comment:
I guess you could be a bit more descriptive here. If I understand the changes correctly, this struct is for storing runtime-specific information, that isn't available at build-time, and therefore can't be made const, correct?